src/Pages/Main/Components/Footer/Footer.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "../Footer/Footer.scss";
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <div className="footer">
+        <section className="footer_top">
+          <button>EnterCloud</button>
+          <div>
+            <ul>
+              <li>블로그</li>
+              <li>이용약관</li>
+              <li>개인정보처리방침</li>
+              <li>운영정책</li>
+              <li>고객 문의</li>
+            </ul>
+          </div>
+        </section>
+        <section className="footer_mid">
+          <ul className="footer_mid_ul">
```

Comment:
우리는 이제 sass의 nesting 이용해서 선택자가 분리되기 때문에 부모태그의 className을 작성하면서 길게 naming을 할 필요가 없습니다! 더불어 className이 필요하지 않을 때는 굳이 사용하지 않아도 되겠네요! 
```suggestion
        <section className="mid">
          <ul>
```